### ISSUE TITLE
Create Image Feature Pipeline

### ISSUE THREAD

**Issue by Jonathan-Pearce** [Jonathan-Pearce/maxsold#16]
Develop an image feature pipeline for our auction item images. This will need to utilize a pre-trained (on ImageNet) CNN model (I would suggest MobileNetV2 or ShuffleNetV2). I would like the code to be developed using PyTorch. 

For the pre-trained model, remove the final layer of the model and simply save the vector of values (features). We will utilize these extracted image features later on to train a regression model.

Utilize the images in the folder data/images for testing
